examples/locality-load-balancing/verify.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash -e
+
+export NAME=locality-load-balancing
+export DELAY=5
+
+# shellcheck source=examples/verify-common.sh
+. "$(dirname "${BASH_SOURCE[0]}")/../verify-common.sh"
+
+check_health() {
+    docker-compose exec -T client-envoy curl -s localhost:8001/clusters | grep health_flags
+}
+
+request_backend() {
+    docker-compose exec -T client-envoy python3 client.py http://localhost:3000/ 100
+}
+
+bring_up_backend() {
+    local server
+    server="$1"
+
+    docker-compose exec -T client-envoy curl -s "$server":8000/healthy
+}
+
+bring_down_backend() {
+    local server
+    server="$1"
+
+    docker-compose exec -T client-envoy curl -s "$server":8000/unhealthy
+}
+
+run_log "=== Demo setup
+client  -> backend-local-1      [priority: 0, weight: 1]
+        -> backend-local-2      [priority: 1, weight: 1]
+        -> backend-remote-1     [priority: 1, weight: 1]
+        -> backend-remote-2     [priority: 2, weight: 1]
+"
+
+run_log "=== Scenario 1: one replica in the highest priority locality"
+
+run_log "Send requests to backend."
+check_health
+request_backend
+
+run_log "Bring down backend-local-1 then snooze for ${DELAY}s. Priority 0 locality is 0% healthy."
+bring_down_backend "${NAME}"_backend-local-1_1
+sleep ${DELAY}
```

Comment:
you can probs make these delays = 1

Reply:
We set the no_traffic_interval to 4s in the client side Envoy config. 5s is necessary to make sure the health check success/fail as we need. Otherwise the demo will fail.

Reply:
ah, k - i was just thinking it would speed up, but defo lets keep as is if its required